pages/api/revalidate.js

```diff
@@ -0,0 +1,17 @@
+export default async function handler(req, res) {
+  // Check for secret to confirm this is a valid request
+  if (
+    !req.query.secret ||
+    !process.env.MY_SECRET_TOKEN ||
+    req.query.secret !== process.env.MY_SECRET_TOKEN
```

Comment:
Silly question but where do these secrets come from?

Reply:
Reading the docs because we have the time limit, I don't think we need the api to invalidate it? or am I missing something?

https://nextjs.org/docs/pages/building-your-application/data-fetching/incremental-static-regeneration#using-on-demand-revalidation

Reply:
Ok thank you for explaining, I will update to use the existing secret we will use for the loading of the data `LINKFREE_API_SECRET`

Then we can merge